### PR TITLE
Unit test node js compiler

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java [hunk 1]
```diff
@@ -24,4 +14,14 @@ public void nodeJsTest() throws CompilerException {
         nodeJsCompiler.nodeCompiler(null);
     }
 
+    @Test
+    public void shouldCompileAJavaFile() throws CompilerException {
+        String filePath = System.getProperty("user.dir")
+                + "/src/test/resources/CompilersResources/NodeJs/test.js";
```
Comment (GustavoZHA) [inline src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java]: The line 19 plus line 20 can be  less 120 characters.

File: src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
- * Information and shall use it only in accordance with the terms of the
- * license agreement you entered into with Fundacion Jala
- *
- * @author Jorge Rodrigo Caceres Velasco
- */
 package org.fundacion.jala.converter.core;
 
 import org.fundacion.jala.converter.core.exceptions.CompilerException;
+import org.fundacion.jala.converter.core.parameter.NodeJsParameter;
 import org.junit.Test;
```
Comment (edsonarios) [inline src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java]: This blanck line should be erased

**Review by edsonarios — APPROVED** [review]

File: src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 public class JavaCompilerTest {
 
-    @Test(expected = CompilerException.class)
+    @Test(expected = NullPointerException.class)
```
Comment (GustavoZHA) [inline src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java:10]: Maybe change NullPointerException to CompilerException, this change don't break unit test

File: src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
```
Comment (GustavoZHA) [inline src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java:7]: Assert.* change to more specific

File: src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java (hunk 3 above)
Comment (JorgeRCV) [reply src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java:10]: If I made that change unit test will break, I will change this to CompilerException when we all finish our unit tests

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**